### ISSUE TITLE
Collectives should be able to generate gift cards

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#1546]

**Comment by lsmith77** [comment]
might be related to leveraging virtual credit cards as a payout option? https://stripe.com/issuing

**Comment by lsmith77** [comment]
ah nevermind .. I guess gift cards are a way to allow people to donate to collectives and not about payout.

**Comment by lsmith77** [comment]
can you explain what this is about?

one thing I would like to with my collective is to have one “main” collective where we collect scheduled donations and one time donations and then sub-collectives for a specific cause where we only do one-off donations usually. money can flow from the “main” collective to a sub-collective (ie. to help bring a cause over the finish line) or vice versa (in case of surplus donations).

what I would like love to do is to allow donors to “move” their donations to a cause on their own to show support. would gift cards and this ticket help here? ie. we would generate gift cards for donors who could then allocate the funds to sub-collections?

now as I am typing out it .. I guess this would be a way too complicated solution.

**Comment by lsmith77** [comment]
ok figured out a better solution since its possible to move money between a collective and sub-collectives without fees as long as they are on the same host 
https://github.com/opencollective/opencollective/issues/1587

**Comment by Betree** [comment]
Yes @lsmith77 you can do collective to collective transfers. This issue is about gift cards but we'll be happy to help you on [Slack](https://slack.opencollective.com/) for specific use cases.

**Comment by Betree** [comment]
I don't see any blockers for this one. @znarf should we activate it?

**Comment by Betree** [comment]
[JHipster](https://opencollective.com/generator-jhipster) is [asking](https://opencollective.freshdesk.com/a/tickets/44880) for this:

> - Use the project's OpenCollective money to create 20 gift cards
> - Send those gift cards electronically to some people
> - Those people would then be able to use that money to sponsor the other projects they like, or maybe even take that money for themselves
